qutebrowser/mainwindow/tabwidget.py

```diff
@@ -136,18 +136,31 @@ def update_tab_title(self, idx, field=None):
                 (fmt is None or ('{' + field + '}') not in fmt)):
             return
 
+        def right_align(text):
+            return str(text).rjust(len(str(self.count())))
+
+        def left_align(text):
+            return str(text).ljust(len(str(self.count())))
+
+        bar = self.tabBar()
+        cur_idx = bar.currentIndex()
+        if idx == cur_idx:
+            rel_idx = left_align(idx + 1) + " "
```

Comment:
Why align all indexes to the right except the current tab's?

Reply:
That is how it looks in vim and vi by default:

![image](https://user-images.githubusercontent.com/13348080/127756567-3cef4ca6-e9f9-4668-84c4-fde57b21038d.png)

Reply:
I suppose if the tabs are at the top, there is no need to align them.

Reply:
For reference this is what it looks like when the tabs are on the side:

![image](https://user-images.githubusercontent.com/13348080/127763904-b7b98df1-e936-41e9-80ff-c6d39f71f1db.png)

Reply:
Ah, I see, it didn't make much sense with the tabs on top. Well, I really like the idea of the relative indexes but I'm not sure about that alignment thing. It's like we have two types of alignment for absolute indexes, but for relative indexes we have just one and it's different from the other two. Just my personal opinion though, I'm not asking you to change anything just yet. Let's see what The-Compiler thinks.

Reply:
I see what your saying. We could have an `aligned_relative_index` in addition to `relative_index`.  

Users probably only care about alignment when they have tabs on thes side. Another option would be to get rid of `aligned_index` and automatically align the indexes when `tabs.position` is set to `left` or `right`.